packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -432,11 +432,15 @@ export function getArrayDefaults<T = any, S extends StrictRJSFSchema = RJSFSchem
 ): T | T[] | undefined {
   const schema: S = rawSchema;
 
-  const neverPopulate = experimental_defaultFormStateBehavior?.arrayMinItems?.populate === 'never';
-  const ignoreMinItemsFlagSet = experimental_defaultFormStateBehavior?.arrayMinItems?.populate === 'requiredOnly';
+  const arrayMinItemsStateBehavior = experimental_defaultFormStateBehavior?.arrayMinItems;
+  const arrayMinItemsPopulate = arrayMinItemsStateBehavior?.populate;
+  const arrayMergeExtraDefaults = arrayMinItemsStateBehavior?.mergeExtraDefaults;
```

Comment:
consider doing this?:
```suggestion
  const { arrayMinItems: arrayMinItemsStateBehavior = {} } = experimental_defaultFormStateBehavior;
  const { populate: arrayMinItemsPopulate, mergeExtraDefaults: arrayMergeExtraDefaults } = arrayMinItemsStateBehavior;
```

Reply:
It's not possible to destructure `experimental_defaultFormStateBehavior` since it can be `undefined` but I improved it a little.